--- ppsspp/CMakeLists.txt.orig	2025-08-16 22:06:04.089557039 +1000
+++ ppsspp/CMakeLists.txt	2025-08-16 22:16:21.606913149 +1000
@@ -2931,11 +2931,18 @@
 		add_executable(${TargetBin} ${NativeAppSource})
 	endif()
 	if(ANDROID AND ARM64)
 		target_link_options(${TargetBin} PRIVATE "-Wl,-z,max-page-size=16384")
 	endif()
-	target_link_libraries(${TargetBin} ${LinkCommon} Common)
+	target_link_libraries(${TargetBin}
+		${LinkCommon}
+		Common
+		SDL2
+		SDL2_ttf
+		${CMAKE_SYSROOT}/usr/lib/libEGL.so
+    	${CMAKE_SYSROOT}/usr/lib/libGLESv2.so
+	)
 endif()
 
 # installs
 if(NOT ANDROID)
 	file(INSTALL ${BigFontAssets} DESTINATION assets)
